### ISSUE TITLE
Warning in Ubuntu

### ISSUE THREAD

**Issue by JJ** [jonathanstowe/Raku-Sys-Lastlog#3]
And this is a warning in Ubuntu:
```
[Sys::Lastlog] Saw 2 occurrences of deprecated code.
[Sys::Lastlog] ================================================================================
[Sys::Lastlog] Sub get_user_by_uid (from System::Passwd) seen at:
[Sys::Lastlog]   /home/jmerelo/.zef/store/Sys-Lastlog-0.0.11.tar.gz/Sys-Lastlog-0.0.11/lib/Sys/Lastlog.pm (Sys::Lastlog), line 207
[Sys::Lastlog] Please use get-user-by-uid instead.
[Sys::Lastlog] --------------------------------------------------------------------------------
[Sys::Lastlog] Method pull-one (from Rakudo::Iterator::Gather) seen at:
[Sys::Lastlog]   SETTING::src/core.c/Iterator.pm6, line 55
[Sys::Lastlog] Please use get-user-by-uid instead.
[Sys::Lastlog] --------------------------------------------------------------------------------
[Sys::Lastlog] Please contact the author to have these occurrences of deprecated code
[Sys::Lastlog] adapted, so that this message will disappear!

```

**Comment by jonathanstowe** [comment]
Hahaha, I made that change too.

However the 

```
[Sys::Lastlog] Method pull-one (from Rakudo::Iterator::Gather) seen at:
[Sys::Lastlog]   SETTING::src/core.c/Iterator.pm6, line 55
[Sys::Lastlog] Please use get-user-by-uid instead.
```

**Comment by jonathanstowe** [comment]
All fixed with v0.0.12  thanks.
